### PR TITLE
ENG-310 lender portfolio

### PR DESCRIPTION
**Pull request by Connorbelez**
No material ENG-310 contract gaps showed up in the changed portfolio code. The core obligations are present in convex/schema.ts (line 465), convex/portfolio/snapshots.ts (line 157), convex/portfolio/snapshots.ts (line 352), convex/crons.ts (line 45), convex/portfolio/history.ts (line 43), convex/portfolio/export.ts (line 104), and convex/portfolio/queries.ts (line 47). The focused tests also cover the required edge cases, including the months: 1 regression path, in convex/portfolio/tests/snapshots.test.ts:171 (line 171) and convex/portfolio/tests/export.test.ts:58 (line 58).

### CODE REVIEW

File: specs/ENG-310/chunks/chunk-03-tests-validation-and-audit/status.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- T-910: Focused portfolio tests passed.
+- T-980: Ran the spec-compliance audit against the current branch diff.
+- T-990: Recorded the audit verdict and blocker state in `specs/ENG-310/audit.md`.
+
+## Validation
+- `bunx convex codegen`: passed
+- `bun check`: passed
+- `bun typecheck`: passed
+- `bun run test -- convex/portfolio/__tests__/snapshots.test.ts convex/portfolio/__tests__/export.test.ts convex/portfolio/__tests__/queries.test.ts`: passed
+- `python3 /Users/connor/.codex/skills/linear-implement-v2/scripts/validate_execution_artifacts.py ENG-310 --repo-root "/Users/connor/.codex/worktrees/a335/fairlendapp" --stage final --require-audit --require-all-tasks-closed --require-all-checklist-closed`: pending rerun after artifact closeout
```
[the remark this answers is not in the document]
Comment (Connorbelez) [reply specs/ENG-310/chunks/chunk-03-tests-validation-and-audit/status.md]: **Fix:** Replaced the machine-specific `python3` invocation and `--repo-root` path with portable placeholders (`<path-to-validate_execution_artifacts.py>`, `<repo-root>`) in `specs/ENG-310/chunks/chunk-03-tests-validation-and-audit/status.md` so closeout instructions work for any checkout (and avoid leaking workstation layout).

```diff
- python3 /Users/connor/.codex/skills/...
+ python3 <path-to-validate_execution_artifacts.py> ... --repo-root "<repo-root>"
```
